cscs-checks/system/nvidia/nvidia_smi_check.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2016-2021 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+import reframe.utility.typecheck as typ
+
+
+@rfm.simple_test
+class nvidia_smi_check(rfm.RunOnlyRegressionTest):
+    gpu_mode = parameter(['accounting', 'compute', 'ecc'])
+    valid_systems = ['daint:gpu', 'dom:gpu']
+    valid_prog_environs = ['builtin']
+    executable = 'nvidia-smi'
+    executable_opts = ['-a', '-d']
+    num_tasks = 0
+    num_tasks_per_node = 1
+    exclusive = True
+    tags = {'maintenance', 'production'}
```

Comment:
@victorusu Flexible tests in maintenance and production? Nah...

Reply:
I would make it `diagnostic`.

Reply:
I talked internally with Matteo and he told me that 1 node is enough to test on maintenance, due to the use of the same image.